stable/mongodb/values-production.yaml

```diff
@@ -0,0 +1,184 @@
+image:
+  ## Bitnami MongoDB registry
+  ##
+  registry: docker.io
+  ## Bitnami MongoDB image name
+  ##
+  name: bitnami/mongodb
```

Comment:
Why did you change this parameter name? By the way, it's not aligned with the README.md